### ISSUE TITLE
SyntaxError when use import module such as cuckoo, androguard

### ISSUE THREAD

**Issue by cmderpt** [VirusTotal/yara-python#75]
Hi,
When I try to run yara on commandline, everything is ok but when I use `yara-python`, I recieved errors:
```python
>>> import yara
>>> rules = yara.compile('Android_ASSDdeveloper.yar')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
yara.SyntaxError: Android_ASSDdeveloper.yar(22): invalid field name "certificate"
```
Android_ASSDdeveloper.yar use [androguard-yara](https://github.com/Koodous/androguard-yara). And I had copy androguard.c into libyara/modules then do follow guide.

Another yara rule use cuckoo module throw error when I run by `yara-python`
```python
>>> import yara
>>> rules = yara.compile('MALW_AZORULT.yar')
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
yara.SyntaxError: MALW_AZORULT.yar(23): invalid field name "sync"
```
I had run `./configure --enable-cuckoo --enable-magic --enable-dotnet` when compile yara.

Those rules are taken from https://github.com/Yara-Rules/rules

I don't know where was I wrong? :(

**Comment by cmderpt** [comment]
Oh sorry, may be I found out my mistake. `source='import module'` parameter in compile method :D

**Comment by nanshihui** [comment]
@cmderpt  Do you install yara-python with --dynamic-linking? how did you install yara with androguard

**Comment by cmderpt** [comment]
My mistake is don't install with --dynamic-linking
But this is error when I try install with --dynamic-linking
```
$sudo python setup.py install --dynamic-linking
usage: setup.py [global_opts] cmd1 [cmd1_opts] [cmd2 [cmd2_opts] ...]
   or: setup.py --help [cmd1 cmd2 ...]
   or: setup.py --help-commands
   or: setup.py cmd --help

error: option --dynamic-linking not recognized
```

**Comment by cmderpt** [comment]
I resolved this issue in #76
